de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/dialogs/ToolAdminDetailsDialog.java

```diff
@@ -94,24 +86,9 @@ public void show(final Tool tool, Mode mode) {
 
     @Override
     public void show() {
-        final ToolContainer toolContainer = factory.getContainer().as();
-        toolContainer.setDeviceList(Lists.<ToolDevice>newArrayList());
-        toolContainer.setContainerVolumes(Lists.<ToolVolume>newArrayList());
-        toolContainer.setContainerVolumesFrom(Lists.<ToolVolumesFrom>newArrayList());
-        ToolImage image = factory.getImage().as();
-        image.setName(appearance.defaultImgPrefix());
-        toolContainer.setImage(image);
-
-        final ToolImplementation toolImplementation = factory.getImplementation().as();
-        final ToolTestData toolTestData = factory.getTest().as();
-        toolTestData.setInputFiles(Lists.<String>newArrayList());
-        toolTestData.setOutputFiles(Lists.<String>newArrayList());
-        toolImplementation.setTest(toolTestData);
```

Comment:
Yup saving fails which is annoying because we have to set all the empty as defaults. In that case do you mind If we the keep the original method instead using `default` method? I see no major improvement with using the `default` method.